controllers/volumereplicationgroup_controller.go

```diff
@@ -61,7 +61,7 @@ type PVUploader interface {
 }
 
 type PVDeleter interface {
-	DeletePV(v interface{}, s3ProfileName string, pvc *corev1.PersistentVolumeClaim) error
+	DeletePV(v interface{}, s3ProfileName string) error
```

Comment:
(nit) DeletePV name should probably change to DeletePVs now that it is deleting a bunch.

Reply:
True.. will make this change.

Reply:
done!

Reply:
Should there be a function to handle deletion of an individual PV from S3 store?

Reply:
I am not sure if necessary, but all the plumbing is there, so if needed, we'll add one.

Reply:
ACK